### PR TITLE
feat: update potential duplicate docs

### PR DESCRIPTION
**Pull request by lucaCambi77**
Update potential duplicate docs based on endpoint updates

- [DHIS2-11336](https://jira.dhis2.org/browse/DHIS2-11336)
- [DHIS2-11338](https://jira.dhis2.org/browse/DHIS2-11338)
- [DHIS2-11532](https://jira.dhis2.org/browse/DHIS2-11532)
- [DHIS2-11437](https://jira.dhis2.org/browse/DHIS2-11437)

### CODE REVIEW

File: src/developer/web-api/tracker.md [hunk 1]
```diff
@@ -2152,9 +2152,9 @@ as shown:
 
 Potential duplicates are records we work with in the data deduplication feature. Due to the nature of the deduplication feature, this API endpoint is somewhat restricted.
 
-A potential duplicate represents a single or pair of records which are suspected to be a duplicate.
+A potential duplicate represents a pair of records which are suspected to be a duplicate.
```
Comment (enricocolasante) [inline src/developer/web-api/tracker.md:2155]: Just for consistency I would always go with Potential Duplicate or potential duplicate

**Review by enricocolasante — APPROVED** [review]
Minor comment

File: src/developer/web-api/tracker.md [hunk 2]
```diff
@@ -2168,15 +2168,42 @@ You can retrieve a list of potential duplicates using the following endpoint:
...
 
     GET /api/potentialDuplicates/<id>
 
+| Error | Description
+|---|---|
+| 404 | Potential duplicate not found
+
+You can also inspect potential duplicates by Tracked Entity Instance (referred as tei) :
+
+    GET /api/potentialDuplicates//tei/<tei>
```
Comment (vilkg) [inline src/developer/web-api/tracker.md]: single "/ "

File: src/developer/web-api/tracker.md [hunk 3]
```diff
@@ -2168,15 +2168,42 @@ You can retrieve a list of potential duplicates using the following endpoint:
...
+
+You can inspect individual potential duplicate records:
 
     GET /api/potentialDuplicates/<id>
 
+| Error | Description
+|---|---|
+| 404 | Potential duplicate not found
+
+You can also inspect potential duplicates by Tracked Entity Instance (referred as tei) :
```
Comment (vilkg) [inline src/developer/web-api/tracker.md]: I would suggest changing 'inspect' here to 'filter'.

File: src/developer/web-api/tracker.md [hunk 4]
```diff
@@ -2185,13 +2212,42 @@ The payload you provide needs at least _teiA_ to be a valid tracked entity insta
 }
 ```
 
-You can mark a potential duplicate as _invalid_ to tell the system that the potential duplicate has been investigated and deemed to be not a duplicate. To do so you can use the following endpoint:
+| Error | Description
+|---|---|
+| 400 | Input teiA or teiB is null or has invalid id
+| 403 | User do not have access to read teiA or teiB
+| 404 | Tei not found
+| 409 | Pair of teiA or teiB already existing
```
Comment (vilkg) [inline src/developer/web-api/tracker.md]: Pair of teiA **and** teiB already existing?

**Review by vilkg — COMMENTED** [review]
Looks great! I left some minor comments :) I would also change "Error" columns in all tables to "Status code". In other places, we use Error to show error codes.

**Review by vilkg — APPROVED** [review]

**Pull request merged**